csharp/Microsoft.Azure.Databricks.Client/Microsoft.Azure.Databricks.Client.csproj

```
@@ -4,6 +4,7 @@
     <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Please disable the code documentation warnings for now. I'll fix them later.
```suggestion
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
    <NoWarn>1570,1573,1587,1591</NoWarn>
```

---

Good call 👍 
I didn't add this because I thought disabling warnings would be over-reaching for a first-time contributor 😄